lms/djangoapps/courseware/features/lti.feature

```diff
@@ -110,7 +110,7 @@ Feature: LMS.LTI component
   Then I see LTI component feedback with text "This is awesome."
   And the LTI provider deletes my grade and feedback
   And I visit the LTI component (have to reload)
-  Then I see LTI component progress with text "(10.0 points possible)"
+  Then I see LTI component progress with text "(0.0 / 10.0 points)"
```

Comment:
But somehow, that changes the calculus of what gets displayed on the LTI progress? Does LTI expect the points values to be some weird Heisenbergian `None` value that *can* indicate a reset, but only sometimes?

I'm going to keep investigating and figure out why this PR's changes have affected this test at all, that seems odd to me.